examples/eks/fargateprofile.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: FargateProfile
+metadata:
+  name: my-fargateprofile
+  labels:
+    examplelabelkey: "examplelabelval"
+spec:
+  forProvider:
+    region: eu-central-1
```

```suggestion
    region: us-east-1
```

Would you mind changing this to `us-east-1`? Most of the examples are in that region, which makes it easier to reference each other.

---

done